modules/azure/version.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    azurerm    = "~> 2.66.0"
```

Comment:
This is going to lock to 2.66.X, could you put `~> 2.0`

Reply:
Yes, of course. I fix it now.